internal/server/bigtable.go

```diff
@@ -52,6 +52,22 @@ func bigTableReadRowsParallel(
 		return nil, nil
 	}
 
+	result := map[string]interface{}{}
+	if getToken == nil {
+		getToken = util.KeyToDcid
+	}
+
+	if btTable == nil {
+		for _, key := range rowList {
+			token, err := getToken(key)
+			if err != nil {
+				return nil, err
+			}
+			result[token] = nil
+		}
+		return result, nil
+	}
```

Comment:
should we add some e2e test for this mode? (later PR is ok)

Reply:
Added unittest and e2e test